packages/next-on-pages/src/buildApplication/fixPrerenderedRoutes.ts

```diff
@@ -109,8 +109,7 @@ async function getRouteDest(
 
 	// Check if a static file already exists at the new location.
 	if (await validateFile(destFile)) {
-		cliWarn(`Prerendered file already exists for ${destRoute}`);
-		return null;
+		cliWarn(`Prerendered file already exists for ${destRoute}, overwriting...`);
```

Comment:
I think this is valid...

it's a bit annoying that with `--skip-build` I can get this sort of incorrectly... but I am not sure if there's anything we can do about this 😓

Reply:
well the message is technically correct, the prerendered file does already exists in the output directory as a static asset, so it's not the end of the world :p